stable/prometheus/templates/alertmanager-networkpolicy.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Values.networkPolicy.enabled }}
+apiVersion: {{ .Values.networkPolicy.apiVersion }}
```

Comment:
Let's take this off users' shoulders and use Helm's `.Capabilities` object to set the API version accordingly.